client/src/app/pages/main-feeds/Feeds.tsx

```diff
@@ -1,13 +1,51 @@
 import React from "react";
 import Header from './header/Header';
-import Feed from './feed/Feed';
+import Post from './post/Post';
+import Sidebar from "./sidebar/Sidebar";
+import Upload from "./upload/Upload";
+import { getCookie } from "../../utils/CookieUtil";
+import './Feeds.scss';
+
 
 export default function Feeds() {
+  const userNameCookie = getCookie("userName");
+  const imageUrlCookie = getCookie("imageUrl");
   return (
   <div className="feeds">
-    <Header />
+    <Header avatarURL={imageUrlCookie} username={userNameCookie} />
       <div className="body">
```

Comment:
don't use className with no meaning name like this, please use something like body-container